README.md

```diff
@@ -83,3 +83,13 @@ jobs:
   deployment.
 - `tests_failed` {int32} - number of mabl tests that failed against this
   deployment.
+
+
+## Contributing
+
+See [here](CONTRIBUTING.md) for details on contributing to this action.
+
+## License
+
+The Dockerfile and associated scripts and documentation in this project are
+released under the [MIT License](LICENSE).
```

Comment:
@gcooney added license section for consistency with our other action, but the LICENSE file was always present, so this isn't a licensing change.